rust_dev_preview/cross_service/rest_ses/src/client.rs

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+//! Application logic aware wrappers around AWS SDK Clients.
+//!
+//! These wrappers add default ARNs for resources the application accesses, and adds those to common calls.
+use std::option::Option;
+
+use aws_config::SdkConfig;
+use aws_sdk_rdsdata::client::fluent_builders::ExecuteStatement;
+use aws_sdk_ses::client::fluent_builders::SendRawEmail;
+use mail_builder::headers::address::Address;
+use secrecy::{ExposeSecret, Secret};
+use serde::Deserialize;
+
+use crate::configuration::{RdsSettings, SesSettings};
+
+/// An AWS RDSData Client, with additional global request information.
+/// The configured db, cluster, and secret manager will be used when executing statements or transactions from this Client.
```

Comment:
For "db, cluster, and secret manager", are these a "DB instance, cluster, and AWS Secrets Manager secret"?